contrib/dracut/90zfs/module-setup.sh.in

```diff
@@ -86,7 +87,7 @@ install() {
 	BB=`hostid | cut -b 3,4`
 	CC=`hostid | cut -b 5,6`
 	DD=`hostid | cut -b 7,8`
-	printf "\x${DD}\x${CC}\x${BB}\x${AA}" > "${initdir}/etc/hostid"
+	echo -ne"\\x${DD}\\x${CC}\\x${BB}\\x${AA}" > "${initdir}/etc/hostid"
```

Comment:
Looks like your missing a space after `echo -ne`

Reply:
Thanks for catching that, I've forced-pushed that commit.